tests/helpers/helpers.ts

```diff
@@ -18,6 +19,18 @@ const maxX = 3;
 const maxY = 4;
 const pvPath = config.get<string>('paths.pvPath');
 const basePath = config.get<string>('paths.basePath');
+const linksPattern = [
```

Comment:
Don't use hard-coded values

Reply:
It was like that in the ingestion catalog.

Reply:
Using random values is recommended. We should change the catalog too.

Reply:
use random sentece ?

Reply:
You need to create a random link. How do you that, it is up to you.